ios/StripeSdk.swift

```diff
@@ -472,6 +472,13 @@ class StripeSdk: RCTEventEmitter, STPApplePayContextDelegate, STPBankSelectionVi
         let merchantIdentifier = self.merchantIdentifier ?? ""
         let paymentRequest = StripeAPI.paymentRequest(withMerchantIdentifier: merchantIdentifier, country: country, currency: currency)
         
+        if (params["capabilityCredit"] as? Bool ?? false) {
+          paymentRequest.merchantCapabilities.update(with: .capabilityCredit)
+        }
+        if (params["capabilityDebit"] as? Bool ?? false) {
+          paymentRequest.merchantCapabilities.update(with: .capabilityDebit)
+        }
+        
```

Comment:
what do you think of something like

```
        if let cardType = params["supportedCardType"] as? NSString {
            if (cardType.isEqual(to: "Credit")) {
                paymentRequest.merchantCapabilities.update(with: .capabilityCredit)
            } else if (cardType.isEqual(to: "Debit")) {
                paymentRequest.merchantCapabilities.update(with: .capabilityDebit)
            }
        }
  ```

So we only add 1 param to handle this, and default to accepting both credit and debit?